### ISSUE TITLE
Notifications page has a <main> inside of a <main>

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14879]
**Describe the bug**

On a user's notification page we can see an error in the axe devtools report, due to a `<main>` element being nested inside of another `<main>` element.

**To Reproduce**

- Log in to the app
- Visit `/notifications`
- Run the axe browser extension (Chrome or Firefox) and observe the error in the console

**Expected behavior**

- These errors shouldn't be present
- There should only be one main landmark on the page
- Visually there should be no change to the layout

**Screenshots**

![screenshot of the notifications page, with axe dev tools report errors highlighted](https://user-images.githubusercontent.com/20773163/135636194-9a103878-2f89-4ac7-8548-3f538b60fc4b.png)

**Comment by luhova** [comment]
Hello, I would like to work on this issue as part of my Hacktoberfest contribution this year. :raised_hands:

**Comment by aitchiss** [comment]
Brilliant! Assigned that to you now @luhova - thanks for taking it on!

**Comment by LAKSHMANANR25** [comment]
Hey, I would also like to take a try on this issue. Can i?

**Comment by aitchiss** [comment]
Hi @LAKSHMANANR25 this issue is already assigned and being worked on - check out our other unassigned issues and let us know if there's something else you'd like to tackle 🙂
